install_config/aggregate_logging.adoc

```diff
@@ -533,7 +530,8 @@ this mode, their saved queries, visualizations, and dashboards are not shared.
 
 You may also set the value `shared_ops`. In this mode, all operations users
 share a Kibana index which allows each operations user to see the same
-queries, visualizations, and dashboards.
+queries, visualizations, and dashboards. Operations users are those users 
+who can satisfy the `SubjectAccessReview`. 
```

Comment:
... ` oc auth can-i get pod/logs -n default`

Reply:
@mburke5678 Maybe we drop the ref to `SubjectAccessReview` and just define the command.  It is something they can run to determine there role.

Reply:
@jcantrill What does the user do if the command shows:
```
#oc auth can-i get pod/logs -n default
no
```
Does the user need `cluster-admin` and `cluster-reader`? You seem unsure in the BZ. What can we tell users to do?

Reply:
They need an administrator to grant those permissions which is typically some cluster admin role.  I don't know if we want to define the here or not.  It implies elevated permissions